test/unit/api/props-with-symbol.js

```diff
@@ -0,0 +1,73 @@
+import afterMutations from '../../lib/after-mutations';
+import hasSymbol from '../../lib/after-mutations';
+import createSymbol from '../../../src/util/create-symbol';
+import element from '../../lib/element';
+import fixture from '../../lib/fixture';
+import { props } from '../../../src/index';
+
+describe('api/props-with-symbol', () => {
+  if (!hasSymbol()) {
+    return;
+  }
+
+  let elem;
+  const secret = createSymbol('secret');
+  const secret2 = createSymbol('secret');
+
+  beforeEach(done => {
+    elem = new (element().skate({
+      props: {
+        [secret]: {
+          initial: 'secretKey',
+        },
+        [secret2]: {
+          initial: 'secretKey2',
```

Comment:
We test here for two symbols with the same descriptor - this should be sufficient to test that we use the actual Symbols as keys instead of stringifying them (in which case they would be indistinguishable)